app.js

```diff
@@ -13,13 +13,32 @@ app.set('views', path.join(__dirname, 'views'));
 app.use(express.static(path.join(__dirname, 'public')));
 
 // Register the location for handlebars partials here:
+hbs.registerPartials(__dirname + '/views/partials');
 
-// ...
 
 // Add the route handlers here:
 
+app.get('/beers', (req, res) => {
+  punkAPI.getBeers()
+    .then(beersFromApi => {
+      res.render('beers', {beersFromApi});
+      //console.log('Beers from the database: ', beersFromApi)
+    })
+    .catch(error => console.log(error));   
+});
+
 app.get('/', (req, res) => {
```

Comment:
As this is the index, what about placing this route above the "/beers" route?

Reply:
of course, I will share the corrected form with you again.